tasks/main.yml

```diff
@@ -11,6 +11,18 @@
   loop:
     - "{{ ca_openssl_path }}/{{ ca_private_path }}"
     - "{{ ca_openssl_path }}/{{ ca_certs_path }}"
+    - "{{ ca_openssl_path }}/{{ ca_requests_path }}"
+    - "{{ ca_openssl_path }}/{{ ca_newcerts_path }}"
+
+- name: ensure ca database text file exists
```

Comment:
This task is not idempotent right now:

```
* [ca-alpine-edgepy38-ansible-previous] => ansible-role-ca : ensure ca database text file exists
```

I think you can add `modification_time` and `access_time` (both set to `preserve`)  to make it idempotent, from the [docs of file](https://docs.ansible.com/ansible/latest/modules/file_module.html):

```yaml
- name: Touch again the same file, but dont change times this makes the task idempotent
  file:
    path: /etc/foo.conf
    state: touch
    mode: u+rw,g-wx,o-rwx
    modification_time: preserve
    access_time: preserve
```

Reply:
I added the parameters mod_time and access_time, but it still fails the test while for me it works like a charm.
I suppose it is because of the openssl dependency I mentioned before.